jobbergate-docs/src/developer_guide/ci.rst

```diff
@@ -2,4 +2,76 @@
  Continuous Integration
 ========================
 
-TODO: Write this in concert with Felipe
+Jobbergate uses `GitHub actions <https://github.com/omnivector-solutions/jobbergate/actions>`_ as continuous integration tools. They are described in details in this page.
+
+-------------------
+ Quality Assurance
+-------------------
+
+A dedicated GitHub action is included at Jobbergate's repository 
+(`test_on_push.yaml <https://github.com/omnivector-solutions/jobbergate/blob/main/.github/workflows/test_on_push.yaml>`_)
+to run the quality assurance tools for all sub-projects at once.
+It includes unit tests, code coverage, linters, code formatters, and static code checkers,
+that are detailed in the section :any:`Quality Assurance Tools`.
+This action is triggered whenever a new commit is pushed to the ``main`` branch or to a pull request.
+
+-----------------
+ Publish on PyPI
+-----------------
+
+The major components of Jobbergate are published on PyPI, the Python Package Index.
+They are available at:
+
+* `jobbergate-api <https://pypi.org/project/jobbergate-api/>`_
+* `jobbergate-cli <https://pypi.org/project/jobbergate-cli/>`_
+
+The steps to publish them to PyPI are automatized by three GitHub actions,
+that are better described bellow.
+
+^^^^^^^^^^^^^^^^^^^^^
+ Prepare for release
+^^^^^^^^^^^^^^^^^^^^^
+
+This action
+(`prepare_release.yaml <https://github.com/omnivector-solutions/jobbergate/blob/main/.github/workflows/prepare_release.yaml>`_)
+is triggered on a workflow dispatch event and demands some interactions
+to ensure code quality. It takes as arguments:
+
+* The base branch where the action will run from. Notice that the use of the ``main``
```

Comment:
```suggestion
* The base branch from which the action will run. Notice that the use of the ``main``
```

Reply:
solved in [58f2f91](https://github.com/omnivector-solutions/jobbergate/pull/144/commits/58f2f918da90e960892e55f68c4e15734043d298).